tests/src/test/scala/whisk/core/loadBalancer/test/SharedDataServiceTests.scala

```diff
@@ -0,0 +1,113 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package whisk.core.loadBalancer.test
+
+import akka.actor.ActorSystem
+import akka.testkit.{ImplicitSender, TestKit}
+import akka.util.Timeout
+import com.typesafe.config.ConfigValueFactory
+import com.typesafe.config.ConfigFactory
+import org.scalatest._
+import whisk.core.loadBalancer._
+
+import scala.concurrent.duration._
+
+// Define your test specific configuration here
+
+object TestKitConfig {
+  val config = """
+    akka.remote.netty.tcp {
+      hostname = "127.0.0.1"
+      port = 2555
+    }
+    """
+}
+
+class SharedDataServiceTests()
+    extends TestKit(ActorSystem("ControllerCluster", ConfigFactory.parseString(TestKitConfig.config)))
+    with ImplicitSender
+    with WordSpecLike
+    with Matchers
+    with BeforeAndAfterAll
+    with BeforeAndAfterEach {
+
+  override def afterAll {
+    TestKit.shutdownActorSystem(system)
+  }
+
+  "A Shared Data Service" must {
+
+    val port = 2552
+    val config = ConfigFactory
+      .parseString("akka.cluster { seed-nodes = [\"akka.tcp://controller-actor-system@127.0.0" +
+        s".1:$port" + "\"] }")
```

Comment:
This is a strange mixture of concatenation and interpolation. Note there is `"""` to be able to use `"` just as you want to:

```scala
.parseString(s"""akka.cluster { seed-nodes = ["akka.tcp://controller-actor-system@127.0.0.1:$port"] }""")
```

Same applies to the other inline config.